go.mod

```diff
@@ -30,6 +30,6 @@ replace (
 	golang.org/x/net => golang.org/x/net v0.0.0-20210503060351-7fd8e65b6420
```

Comment:
you should not need to use these replace block, replace is for pinning dependencies

Reply:
can you try to remove the entire `replace `block and run `go mod tidy && go mod vendor ` again?

Reply:
_>you should not need to use these replace block, replace is for pinning dependencies_
While I'm not aware of the context of why that was done, but I did see a commit from @spencerhance  to explicitly pin them. Spencer, can you clarify if this is still needed?

_>can you try to remove the entire replace block and run go mod tidy && go mod vendor  again?_
Gave this a shot. It failed with another reason. I believe the explanation in https://github.com/GoogleCloudPlatform/k8s-cloud-provider/pull/89#issuecomment-1485687732 should be the cause for this too.

Reply:
We probably don't need to pin dependencies anymore.  We originally did this when we first migrated to go modules to make sure that dependencies wouldn't get updated.

Reply:
Thanks Spencer! I removed the pinned dependencies.